extra_requirements.txt

```diff
@@ -15,5 +15,5 @@ scandir>=1.8
 # we use the latest version to see any problems with new versions
 pandas==1.3.5; python_version == '3.7' # pyup: ignore
```

Comment:
I think this line should be removed (and the next one updated).

Reply:
I left it in as we still test PyPi 3.7 (which is not end of life yet).